RestaurantMap/Scener/Restaurant/RestaurantInteractor.swift

```diff
@@ -6,9 +6,23 @@
 //
 
 import Foundation
+import UIKit
 
 class RestaurantInteractor: RestaurantBusinessLogics {
     
     var presenter: RestaurantPresentationLogics?
     
+    func callPhoneNumber(_ phoneNumber: String) {
+        if let url = URL(string: "telprompt://\(phoneNumber)"),
+           UIApplication.shared.canOpenURL(url) {
+            UIApplication.shared.open(url, options: [:], completionHandler: nil)
+        }
+    }
+    
+    func openWebsite(_ url: String) {
+        if let url = URL(string: url),
+           UIApplication.shared.canOpenURL(url) {
+            UIApplication.shared.open(url)
+        }
+    }
```

Comment:
This is navigation to another screen or web view or safari, you should put these in Router